priv/templates/smartos/Makefile

```diff
@@ -127,29 +133,29 @@ packing_list_files: $(BUILD_STAGE_DIR) templates
 	   echo "@comment Packing /opt/local/etc files" >> +CONTENTS && \
 	   echo "@cwd /opt/local" >> +CONTENTS && \
 	   echo "@owner root" >> +CONTENTS && \
-	   echo "@group {{package_install_user}}" >> +CONTENTS && \
+	   echo "@group riak" >> +CONTENTS && \
```

Comment:
Was this change intentional? I think it should stay as package_install_user since node_package isn't supposed to be riak-specific, right?